lib/home_page.dart

```diff
@@ -0,0 +1,79 @@
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+import 'package:pokedex/features/favorite_list/favorite_connector.dart';
+import 'package:pokedex/features/pokemon_list/pokemon_list_connector.dart';
+import 'package:pokedex/utils/strings.dart' as str;
+import 'package:pokedex/utils/constant.dart' as k;
+
+class HomePage extends StatefulWidget {
+  const HomePage({
+    Key? key,
+    required this.child,
+  }) : super(key: key);
+
+  final Widget child;
+
+  @override
+  State<HomePage> createState() => _HomePageState();
+}
+
+class _HomePageState extends State<HomePage> {
+  @override
+  Widget build(BuildContext context) {
+    final router = GoRouter.of(context);
+    final currentNavIndex = _selectNavIndex(router.location);
+    final navBarItems = <BottomNavigationBarItem>[
+      const BottomNavigationBarItem(
+        label: str.pokedexTitle,
+        icon: ImageIcon(AssetImage(k.homeIcon)),
```

Comment:
instead of importing new common assets for icon, would it be better if you just utilize `Icon` widget then use its varieties of predefined icons via `Icons.[anything you woud want]`? 🙂